specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -6,127 +6,129 @@ This specification refers to the following
+_Jakarta™ JSON Processing Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/jsonp/2.0_
 
-_Jakarta™ JSON Binding Specification, Version 1.0_. Available at: _https://jakarta.ee/specifications/jsonb/1.0_
+_Jakarta™ JSON Binding Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/jsonb/2.0_
 
-_Jakarta™ Concurrency Specification, Version 1.1_. Available at: _https://jakarta.ee/specifications/concurrency/1.1_
+_Jakarta™ Concurrency Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/concurrency/2.0_
 
-_Jakarta™ Batch Specification, Version 1.0_. Available at: _https://jakarta.ee/specifications/batch/1.0_
+_Jakarta™ Batch Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/batch/2.0_
 
 Extension Mechanism Architecture, Available at
-_https://docs.oracle.com/javase/8/docs/technotes/guides/extensions/index.html_.
+_https://docs.oracle.com/javase/8/docs/technotes/guides/extensions/index.html_
 
 Optional Package Versioning, Available at
-_https://docs.oracle.com/javase/8/docs/technotes/guides/extensions/index.html_.
+_https://docs.oracle.com/javase/8/docs/technotes/guides/extensions/index.html_
 
 JAR File Specification, Available at
-_https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html_ .
+_https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html_
 
 _The Common Object Request Broker: Architecture
 and Specification_ (CORBA 2.3.1 specification), Available at
-_https://www.omg.org/cgi-bin/doc?formal/99-10-07_ .
+_https://www.omg.org/cgi-bin/doc?formal/99-10-07_
 
 CORBA 2.6 - Chapter 26 - Secure
 Interoperability, Available at
-_https://www.omg.org/cgi-bin/doc?formal/01-12-30_ .
+_https://www.omg.org/cgi-bin/doc?formal/01-12-30_
 
 _IDL To Java™ Language Mapping Specification_ ,
-Available at _https://www.omg.org/cgi-bin/doc?ptc/2000-01-08_ .
+Available at _https://www.omg.org/cgi-bin/doc?ptc/2000-01-08_
 
 _Java™ Language To IDL Mapping Specification_ ,
-Available at _https://www.omg.org/cgi-bin/doc?ptc/2000-01-06_ .
+Available at _https://www.omg.org/cgi-bin/doc?ptc/2000-01-06_
 
 _Interoperable Naming Service,_ Available at
-_https://www.omg.org/cgi-bin/doc?ptc/00-08-07_ .
+_https://www.omg.org/cgi-bin/doc?ptc/00-08-07_
 
 _Transaction Service Specification_ (OTS
 specification), Available at
-_https://www.omg.org/cgi-bin/doc?formal/2001-11-03_ .
+_https://www.omg.org/cgi-bin/doc?formal/2001-11-03_
 
 _The SSL Protocol, Version 3.0._ Available at
-_https://tools.ietf.org/html/rfc6101_ .
+_https://tools.ietf.org/html/rfc6101_
 
 Architectural Styles and the Design of
 Network-based Software Architectures (REST), R. Fielding, Ph.d
 dissertation, University of California, Irvine, 2000. Available at
-_https://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm_ .
+_https://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm_
 
 Java™ Community Process
```

Comment:
Do we still need this?  If so, should we also point at our Jakarta EE Specification Process?  That might be good.  But, it up near the Jakarta items before the "java extension" items...

Reply:
Good point! I will add a reference to the JESP above. As long as we still have references to JCP specifications, it doesn't hurt to have e ref to the JCP Process as well

Reply:
Ivar, I don't see an update for the JESP reference...

Reply:
Never mind, need more coffee... :-)